### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by ShubhamGhanmode**
How to upload phot?

### CODE REVIEW

**Review by nikit19 — COMMENTED** [review]
Upload the photo in your forked repository in the img/blogs folder

File: _data/blogs.yml [hunk 1]
```diff
@@ -283,3 +283,7 @@
   img: pp.jpeg
   author_name: Nishant S.
 
+- url: https://shubham970121859.wordpress.com/
+  title: Me & Google code-in
+  img: syg.jpg
```
Comment (anshumanv) [inline _data/blogs.yml:288]: Please upload an image with this name.

**Review by codedsun — CHANGES_REQUESTED** [review]
Add your image :)

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
Indent too

**Review by isuruAb — COMMENTED** [review]
please resolve conflicts

**Review by ParthS007 — COMMENTED** [review]
@ShubhamGhanmode  Closing this PR due to many conflicts and inactivity.Please read `README.MD` on how to add a blog and make a PR.Please ask on FOSSASIA Gitter channel if you face any difficulty 👍

**Pull request closed without merging**